features/content_data_admin.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Content Data Admin
+  Tests for the Content Data Admin application, which provides
+  publishers with data about the content they manage
+
+  # TODO: Doesn't work yet, as the Smokey user doesn't have an
```

Comment:
@cbaines what do we have to do to have an organisation for Smokey?

Reply:
> @cbaines what do we have to do to have an organisation for Smokey?

We could set an organisation, I think it's as simple as just making the change in the Signon user interface. The tricky bit is making sure that doesn't have any unexpected effects on the other tests.

Reply:
> We could set an organisation, I think it's as simple as just making the change in the Signon user interface. The tricky bit is making sure that doesn't have any unexpected effects on the other tests.

I've made this change now, and I've updated the comment. Even though it should work now, most of the time the test will still fail due to the slowness of the index page.

Reply:
With the index page speed improvements, I've marked both tests as `@normal` now.